playbooks/roles/backups/defaults/main.yml

```diff
@@ -28,3 +28,6 @@ BACKUPS_GS_SERVICE_KEY_FILE_PASSWORD: 'notasecret'
 # Azure
 BACKUPS_AZURE_STORAGE_ACCOUNT: ''
 BACKUPS_AZURE_STORAGE_KEY: ''
+
+# Prometheus Pushgateway
+PUSHGATEWAY: 'https://pushgateway.infra.appsembler.com'
```

Comment:
@thraxil shouldn't we add a secret key to this API?

Reply:
@OmarIthawi Yeah... having a central pushgateway like that isn't really an ideal setup security-wise. Pushgateway itself doesn't really support any kind of security API; the intention is that you would run it inside your VPC or whitelist access. For us, that would really mean running an instance per project. We could hack in basic auth with nginx I guess as an interim solution. So far it hasn't been a priority, since these metrics aren't sensitive data.

Reply:
I like the nginx auth + HTTPs (which is already there), it looks great to me.